src/app/pages/reports-dashboard/components/exporters/reporting-exporters-form/reporting-exporters-form.component.html

```diff
@@ -0,0 +1,47 @@
+<ix-modal-header [title]="title" [loading]="isLoading"></ix-modal-header>
+
+<mat-card>
+  <mat-card-content>
+    <form
+      class="ix-form-container"
+      [formGroup]="form"
+      (submit)="onSubmit()"
+    >
+      <ix-fieldset>
+        <ix-input
+          formControlName="name"
+          [label]="'Name' | translate"
+          [tooltip]="'Give your exporter configuration a name'"
+          [required]="true"
+        ></ix-input>
+        <ix-select
+          formControlName="type"
+          [label]="'Reporting Exporter Type' | translate"
+          [options]="exporterTypeOptions$"
+          (ngModelChange)="onExporterTypeChanged($event)"
```

Comment:
It's better to use valueChanges with reactive forms. This is kinda non-standard.